project.md

```diff
@@ -55,14 +87,20 @@ This section examines whether the idea makes for a good CS 111 project.
 _If someone were to work on this project, what percentage of their time would be
 spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
-project (e.g., implementing a complicated semantics that doesn't require a lot
+project (e.g., implementing complicated semantics that doesn't require a lot
 of language design)?_
 
+My initial thought was I would say %50/%50 but after further thought, I believe it to be %75/%25. This is based on the fact that I will be writing two DSLs which should give me plenty of opportunity to directly engage in language design. However, this is tailored to the scope of this class assignment, if I wanted to I could add a lot of features that would require me to work with the systems aspects. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I believe that this idea is very doable as a project over the remaining time that we have this semester. There is a chance that this project takes me slightly longer than I am currently expecting it to take but this is ok. Since I will have to start by writing the internal DSLs for my program, and then get the application up and running, I will be able to use this class as a resource to write my DSLs. After that, I can finish the systems aspects of the project on my own time or during the end of the semester if time permits.
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+This is a good idea for a project because it is something I would use, and often. It would allow me to create my DSL and create a product I can hopefully share with others too. It might not be a good idea because it will require me to work a decent amount on the systems side of things which is not the main focus of this class. Also, many people are comfortable and used to using existing scheduling systems. However, I believe the pros of this project far outweigh the cons as I am passionate about working on it, I get the chance to write not one but two DSLs, and it would allow me to practice writing a functional application which is something I've been wanting to gain more experience doing.
```

Comment:
I think the systems aspects can be alleviated by utilizing libraries in Java or Scala, which should free up more time for language design. Designing 2 DSLs intended to work together could mean potential ways to simplify the design of both DSLs.